geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl

```diff
@@ -113,6 +115,11 @@ def main():
     default = not (args.config or args.geoportal)
 
     if default or args.config:
+        simple = not os.path.exists("geoportal/Dockerfile")
+        if args.simple:
+            simple = True
+        if args.not_simple:
+            simple = False
```

Comment:
Is it not necessary to pass this as a build arg to the docker build command for the config image ?

Reply:
Arg +1 :-)